var/spack/repos/builtin/packages/rocm-device-libs/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class RocmDeviceLibs(CMakePackage):
+    """set of AMD specific device-side language runtime libraries"""
+
+    homepage = "https://github.com/RadeonOpenCompute/ROCm-Device-Libs"
+    url      = "https://github.com/RadeonOpenCompute/ROCm-Device-Libs/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='dce3a4ba672c4a2da4c2260ee4dc96ff6dd51877f5e7e1993cb107372a35a378')
+
+    variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+
+    depends_on('cmake@3:', type='build')
+    for ver in ['3.5.0']:
+        depends_on('llvm-amdgpu@' + ver, type='build', when='@' + ver)
```

Comment:
You need to make this a link dependency too, so just write:
```suggestion
        depends_on('llvm-amdgpu@' + ver, type='build', when='@' + ver)
        depends_on('llvm-amdgpu@' + ver, when='@' + ver)
```